public/data/avneesh0612.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Avneesh Agarwal",
+  "bio": "I am a web developer, designer, content creator, open source contributor and student.",
```

Missing comma there. Going to make PR for it. 🙂
```suggestion
  "bio": "I am a web developer, designer, content creator, open source contributor, and student.",
```